pages/common/expr.md

```diff
@@ -3,22 +3,30 @@
 > Evaluate expressions and manipulate strings.
 > More information: <https://www.gnu.org/software/coreutils/expr>.
 
-- Get string length:
+- Get the length of a specific string:
 
-`expr length {{string}}`
+`expr length '{{hello world}}'`
```

Comment:
```suggestion
`expr length '{{string}}'`
```
I think the old placeholder is fine here.

Reply:
Yes, it's definitely better.